src/lib/elements.ts

```diff
@@ -0,0 +1,78 @@
+export type TagName = keyof HTMLElementTagNameMap | 'svg';
```

Comment:
I sometimes find `src` and `lib` interchangeable, which always makes `src/lib` feel odd to me. Or rather, it's nice when a subfolder gives context as to what's inside. 

Since this is a folder with 1 file in it, what do you think about leaving this in `src/elements.ts` and creating a subfolder later down the road?

Reply:
Totally agree, this was blindly copy-pasted from the Svelte plugin, which has multiple files in that folder. But I don't actually think we'll ever need anything else there unlike them